studio/components/interfaces/Home/Home.constants.ts

```diff
@@ -6,19 +6,40 @@ export const CLIENT_LIBRARIES = [
     docsUrl: 'https://supabase.com/docs/reference/javascript/installing',
     gitUrl: 'https://github.com/supabase/supabase-js',
   },
+  {
+    language: 'Flutter',
+    officialSupport: true,
+    releaseState: undefined,
+    docsUrl: 'https://supabase.com/docs/reference/dart/installing',
+    gitUrl: 'https://github.com/supabase/supabase-flutter',
+  },
   {
     language: 'Python',
     officialSupport: false,
     releaseState: 'Alpha',
-    docsUrl: 'https://supabase.com/docs/reference/python/introduction',
+    docsUrl: 'https://supabase.com/docs/reference/python/initializing',
     gitUrl: 'https://github.com/supabase/supabase-py',
   },
   {
-    language: 'Flutter',
-    officialSupport: true,
+    language: 'C-Sharp',
```

Comment:
Does 'C#' work as a label here (to be consistent with docs)?
```suggestion
    language: 'C#',
```